packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -251,6 +251,27 @@ Image createImageIcon(String url, {double size = defaultIconSize}) {
   );
 }
 
+class ThemedImageIcon extends StatelessWidget {
```

Comment:
a lot of the calls to this class have the exact same file for both light mode and dark mode. Create a separate ImageIcon class for that case to improve readability.